key-concepts/architecture.md

```diff
@@ -28,30 +25,41 @@ The GraphQL Server is a separate web server providing a GraphQL API (more on the
 
 ### Squid
 
-A Squid is a query node for ingesting, transforming, and presenting Substrate blockchain data.
+A Squid is a query node for transforming, and presenting Substrate blockchain data. It consists of:
```

Comment:
Query Node may be a bit confusing/unclear here. Maybe ETL (Extract-Transform-Load) pipeline is better here?

Reply:
Sounds good. I have used Query Node for lack of better terms, I think this is where we need to find the right terminology, once and for all and stick to it. ETL is very descriptive, I used it already for the whole framework, so I'm going to give it a try.
Thanks